src/topology/algebra/order/liminf_limsup.lean

```diff
@@ -215,6 +216,32 @@ begin
   exact H a as b bs ab ⟨A, B⟩,
 end
 
+variables [first_countable_topology α] {f : filter β} [countable_Inter_filter f]
+  {u : β → α} {a : α}
+
+lemma eventually_le_of_limsup_le (hfua : f.limsup u ≤ a)
+  (hf : is_bounded_under (≤) f u . is_bounded_default) : ∀ᶠ b in f, u b ≤ a :=
```

Comment:
This looks like a strange formulation. Why not just `∀ᶠ b in f, u b ≤ f.limsup u`?

Reply:
because it's more practical than having to apply monotonicity of `eventually`. In my case, this lemma translates a paper step, but I'm happy to simplify it.